superset-frontend/src/dashboard/components/nativeFilters/FilterBar/Header.tsx

```diff
@@ -98,10 +98,14 @@ const Header: FC<HeaderProps> = ({
         <span>{t('Filters')}</span>
         {canEdit && (
           <FilterConfigurationLink createNewOnOpen={filterValues.length === 0}>
-            <Icon name="edit" data-test="create-filter" />
+            <Icon name="edit" role="button" data-test="create-filter" />
           </FilterConfigurationLink>
         )}
-        <Icon name="expand" onClick={() => toggleFiltersBar(false)} />
+        <Icon
+          name="expand"
+          role="button"
+          onClick={() => toggleFiltersBar(false)}
```

Comment:
Is codecov correct that this is not being tested? If so, should we mock that onClick?

Reply:
I just touched this file to add the button role to the icons. The tests of `FilterBar` are here https://github.com/apache/superset/pull/13916